code-challenges.test.js

```diff
@@ -15,42 +15,174 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
-
-const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+describe("fibaSeq", () => {
+    const fibLength1 = 6
+    // Expected output: [1, 1, 2, 3, 5, 8]
+    
+    const fibLength2 = 10
+     // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+    it("takes a number for an argument and returns an array in fibonaci     sequence equal to the lenght of the number", () => {
+      expect(fibaSeq(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+      expect(fibaSeq(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34,55]    )} 
+    )
+}
+)
+ 
+//Yarn jest result:
+
+//Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+// Snapshots:   0 total
+// Time:        0.487 s
+// Ran all test suites.
+// error Command failed with exit code 1.
 
 // b) Create the function that makes the test pass.
+// Pseudo code:
+// create a function called fibaSeq to generate fibonacci series up to n terms
+
+// 
+const fibaSeq = (number) => {
+let fibaNums = [0, 1]  //to initialize the first two values of the fibonacci sequence
+// next is to run a loop adding the last two values in the sequence to come up with the value of the next value in the sequence, push the next value in the fibaNums array.
+           
+           for(let i = 1; i < (n - 2); i++) {
+               fibaNums.push(fibaNums[fibaNums.length-1] + fibaNums[fibaNums.length- 2])
+           } return (fibaNums)
+       
+   }
+    
+console.log(fibaSeq(fibLength1))
+console.log(fibaSeq(fibLength2))
+
+// Test result:
+//FAIL  ./code-challenges.test.js
+// ● Test suite failed to run
+
+// ReferenceError: fibLength1 is not defined
+
+//   55 |     
+//   56 |
+// > 57 | console.log(fibaSeq(fibLength1))
+//      |                     ^
+//   58 | console.log(fibaSeq(fibLength2))
+//   59 |
+//   60 | // Test result:
+
+//   at Object.fibLength1 (code-challenges.test.js:57:21)
+//   at TestScheduler.scheduleTests (node_modules/@jest/core/build/TestScheduler.js:317:13)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       0 total
+// Snapshots:   0 total
+// Time:        0.381 s
+// Ran
+// End of test result
+// Not sure how to fix the code to make it past the test. I was stuck trying to fix it to make it work. so I proceeded to the next problem.
```

Comment:
You are using your console.log on lines 57 and 58. If you get rid of those lines you will be able to take step forward in figuring out this problem. There are a few items that you will want to work through with the test invoking your function for you.